cmake/alpakaCommon.cmake

```diff
@@ -772,7 +772,8 @@ if(TARGET alpaka)
 endif()
 
 # NVCC does not incorporate the COMPILE_OPTIONS of a target but only the CMAKE_CXX_FLAGS
-if(ALPAKA_ACC_GPU_HIP_ENABLE AND ALPAKA_CUDA_COMPILER MATCHES "nvcc")
+# For hipcc we need to propagate the CMAKE_CXX_FLAGS to HIP_HIPCC_FLAGS.
+if(ALPAKA_ACC_GPU_CUDA_ENABLE OR ALPAKA_ACC_GPU_HIP_ENABLE)
```

Comment:
Is CUDA still required here? Don't we have native CMake support now?

Reply:
mhh thats a very good question 0.7.0-rc1 and the develop PR. I need to test if this part is required for `ALPAKA_ACC_GPU_CUDA_ENABLE`